three-node-cluster/3-node-cluster.yml

```diff
@@ -0,0 +1,659 @@
+# Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Licence year

Reply:
Fixed by https://github.com/wso2/performance-is/pull/128/commits/6f54cb9f8218f1e659a5822fdc7b810124ba65f4